tck/features/CreateAcceptance.feature

```diff
@@ -0,0 +1,590 @@
+
+  Scenario: Creating a node with null properties should not return those properties
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (n {id: 12, property: NULL})
+      RETURN n.id AS id
+      """
+    Then the result should be:
+      | id |
+      | 12 |
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +properties | 1 |
+
+  Scenario: Creating a relationship with null properties should not return those properties
+    Given an empty graph
+    When executing query:
+      """
+      CREATE ()-[r:X {id: 12, property: NULL}]->()
+      RETURN r.id
+      """
+    Then the result should be:
+      | r.id |
+      | 12   |
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +properties    | 1 |
+
+  Scenario: Create a simple pattern
+    Given an empty graph
+    When executing query:
+      """
+      CREATE ()-[:R]->()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+
+  Scenario: Create a self loop
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (root:R)-[:LINK]->(root)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 1 |
+      | +relationships | 1 |
+      | +labels        | 1 |
+
+  Scenario: Create a self loop using MATCH
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:R)
+      """
+    When executing query:
+      """
+      MATCH (root:R)
+      CREATE (root)-[:LINK]->(root)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +relationships | 1 |
+
+  Scenario: Create nodes and relationships
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (a), (b),
+             (a)-[:R]->(b)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+
+  Scenario: Create a relationship with a property
+    Given an empty graph
+    When executing query:
+      """
+      CREATE ()-[:R {prop: 42}]->()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +properties    | 1 |
+
+  Scenario: Create a relationship with the correct direction - 1
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:X)
+      CREATE (:Y)
+      """
+    When executing query:
+      """
+      MATCH (x:X), (y:Y)
+      CREATE (x)<-[:TYPE]-(y)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +relationships | 1 |
+
+  Scenario: Create a relationship with the correct direction - 2
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:X)
+      CREATE (:Y)
+      """
+    When executing query:
```

Comment:
This step should have its own `Then the result should be empty` and `And the side effects should be`